### PR TITLE
create EDA for sample dataset

### PR DESCRIPTION
**Pull request by chz816**

### CODE REVIEW

File: eda/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# EDA
+
```
Comment (harrywang) [inline eda/README.md]: no need for a readme. please remove this file and put the notebook in pre-processing folder.

**Comment by harrywang** [comment]
Screen Shot 2020-06-07 at 4 21 26 PM

print only say first 5 to keep the notebook neat. - same for other similar cells. 


e.g.:

Screen Shot 2020-06-07 at 4 25 59 PM

**Comment by harrywang** [comment]
delete empty cells

**Comment by chz816** [comment]
update README.md and EDA notebook

**Comment by harrywang** [comment]
fixed #4

**Pull request merged**